src/test/java/org/apache/commons/collections4/bag/AbstractBagTest.java

```diff
@@ -125,7 +125,7 @@ public void resetFull() {
 
     //-----------------------------------------------------------------------
     /**
-     * Returns the {@link #collection} field cast to a {@link Bag}.
+     * Returns the {@link Collection} field cast to a {@link Bag}.
```

Comment:
That's a link to the [`collection`](https://github.com/apache/commons-collections/blob/3945e6c48e2dd83c6b4b65c135f0bfc2f28ed52a/src/test/java/org/apache/commons/collections4/collection/AbstractCollectionTest.java#L141) variable of the parent class.

Reply:
sorry..I revert AbstractBagTest

Reply:
No problem, thanks @EruDev . Could you squash your commits too? For a small change, it's preferable to have a single (or very few) commit. Please.

Thanks!

Reply:
I close this PR and resubmit.